src/test/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/resthandler/RestChangePolicyActionIT.kt

```diff
@@ -194,7 +194,7 @@ class RestChangePolicyActionIT : IndexStateManagementRestTestCase() {
         assertNull("Policy has already initialized", managedIndexConfig.policy)
         assertEquals("Policy id does not match", policy.id, managedIndexConfig.policyID)
 
-        // if we try to change policy now, it'll have no ManagedIndexMetaData yet and should succeed
+        // if we try to change policy now, it haven't actually run so have no ManagedIndexMetaData yet, should succeed then
```

Comment:
Can reword to:

"If we try to change the policy now, it hasn't actually run and has no ManagedIndexMetaData yet so it should succeed"

Reply:
reworded